### PR TITLE
Dependency update

### PR DESCRIPTION
**Pull request by Ghnuberath**

### CODE REVIEW

File: lib/db/database.js [hunk 1]
```diff
@@ -101,13 +101,13 @@ TransactionFactory.prototype[sOpenConnections] = function (providerNames, closer
...
         // chain an extra then() on the end, which will store connection closing functions
         // which will allow us to clean up if one of the opens fails.
-        .then((conn) => {
-          closers.push((shouldCommit) => {
-            return p.exitTransaction(conn, shouldCommit);
+          .then((conn) => {
+            closers.push((shouldCommit) => {
+              return p.exitTransaction(conn, shouldCommit);
+            });
+            conn[sConnName] = p.name; // store name in promise for later
```
Comment (mlaccetti) [inline lib/db/database.js]: Was this a bug, that it was set elsewhere?

Comment (Ghnuberath) [inline lib/db/database.js]: Hm? Isn't this just a whitespace change?

Comment (mlaccetti) [inline lib/db/database.js]: Is it? The github diff makes it look like the conn stuff moved around. Ugh.

Comment (Ghnuberath) [inline lib/db/database.js]: I think so!? Haha.

**Pull request merged**